content/en/02-installing-pixie/03-install-guides/03-airgap-pixie.md

```diff
@@ -0,0 +1,281 @@
+  There is a known issue with login on self-managed Pixie Cloud on Safari and Firefox. For now, use Chrome.
+</Alert>
+
+### Serve the Script Bundle
+
+After logging into the Pixie UI using the admin account, you will see a blank UI with the following errors in the DevTools console:
+
+<svg title='' src='airgap/ui-error.png' />
+
+To resolve these errors, you'll need to do one of the following:
+
+- [Setup the script dev environment](/installing-pixie/install-guides/airgap-pixie/#deploy-pixie-cloud-setup-the-script-dev-environment)
+- [Override the `SCRIPT_BUNDLE_URLS` environment variable](https://github.com/pixie-io/pixie/issues/485#issuecomment-1191969058)
+
+#### Setup the Script Dev Environment
+
+To setup the script dev environment:
+
+```
+
+git clone https://github.com/pixie-io/pixie.git
+cd pixie/src/pxl_scripts
+make dev
+
+```
+
+Open Chrome’s DevTools console and run the following:
+
+```
+
+localStorage.setItem('px-custom-oss-bundle-path', 'http://127.0.0.1:8000/bundle-oss.json')
+localStorage.setItem('px-custom-core-bundle-path', 'http://127.0.0.1:8000/bundle-oss.json')
+
+```
+
+Once you have set these variables, do a **soft** reload of the UI webpage (a hard reload will clear the variable you just set).
+
+Once you're able to see Pixie's UI, you'll need to use Pixie's UI to [create a deploy key](/reference/admin/deploy-keys/#create-a-deploy-key-using-the-live-ui). Record this value somewhere, we'll use it in a future step.
+
+### Invite others to your organization (optional)
+
+Add users to your organization to share access to Pixie Live Views, query running clusters, and deploy new Pixie clusters. For instructions, see the [User Management & Sharing](/reference/admin/user-mgmt) reference docs.
+
+## Deploy Pixie
+
+1. Download the vizier artifacts:
+
+```
+curl https://storage.googleapis.com/pixie-dev-public/vizier/latest/vizier_yamls.tar | tar x
+cd yamls
+```
+
+2. Update the `deploy-key` and `PX_CLUSTER_NAME` values in the `vizier/secrets.yaml` file. Remember that you previously created the deploy key in Pixie UI's.
+
+3. Deploy the `vizier/secrets.yaml` file.
+
+```
+kubectl apply -f vizier/secrets.yaml
+```
+
+5. Determine whether you'd like to deploy Pixie with or without `etcd`. We recommend installing Pixie without `etcd` as long as your cluster supports Pixie creating and using PVs.
+
+> To deploy Pixie without `etcd`, use the following yamls:
+>
+> - `vizier/vizier_metadata_persist_prod.yaml`
+> - `vizier_deps/nats_prod.yaml`
+
+> To deploy Pixie with `etcd`, use the following yamls:
+>
+> - `vizier/vizier_etcd_metadata_prod.yaml`
+> - `vizier_deps/etcd_prod.yaml`
+
+6. Collect and publish the required Pixie Vizier images to a private registry. _Note: the below commands assume you are deploying Pixie without `etcd`._
+
+> To list the images needed to deploy Pixie Cloud:
+
+```
+cat images/vizier_image_list.txt
+```
+
+7. Apply the yamls. _Note: the below commands assume you are deploying Pixie without `etcd`._
```

Comment:
Also add the step where they should update the image tags in the YAMLs to point to their private registry.